src/assets/YAML/default/BuildAndDeployment/Build.yaml

```diff
@@ -24,7 +24,7 @@ Build and Deployment:
       level: 2
       implementation:
       - $ref: src/assets/YAML/default/implementations.yaml#/implementations/ci-cd-tools
-      - $ref: src/assets/YAML/default/implementations.yaml#/implementations/container-technologi
```

Comment:
>  I associate Kubernetes with it. For some env. it is just to run the builds in a container 

You're right, but I wanted to avoid the following "flaky" assessment:

- a team uses Docker without any automation mechanisms (no compose, no kube, no cloud run, ...)
- it marks this activity as done

In particular, just using docker is quite weak for the testing phase.